### ISSUE TITLE
Flake8 block comment warning for e.g. `#| echo: false`

### ISSUE THREAD

**Issue by amyheather** [lintquarto/lintquarto#23]
Flake8 gives a warning like:

```
docs/pages/behind_the_scenes.qmd:12:1: E265 block comment should start with '# '
```

For the special quarto comments which are like: `#|`

**Comment by amyheather** [comment]
**Solution:** Implement some pre-processing to amend cell markers so they don't flag linters
(e.g. if after `#%[python]` and starts `#|` then change to `# |`).
